integration_tests/grpc_rest/validator/helpers.go

```diff
@@ -0,0 +1,297 @@
+) (*sdk.TxResponse, error) {
+	msg := validatortypes.NewMsgDisableValidator(valAddr)
+	return suite.BuildAndBroadcastTx([]sdk.Msg{msg}, signerName, signerAccount)
+}
+
+func EnableValidator(
+	suite *utils.TestSuite,
+	valAddr sdk.ValAddress,
+	signerName string,
+	signerAccount *dclauthtypes.Account,
+) (*sdk.TxResponse, error) {
+	msg := validatortypes.NewMsgEnableValidator(valAddr)
+	return suite.BuildAndBroadcastTx([]sdk.Msg{msg}, signerName, signerAccount)
+}
+
+func ProposeDisableValidator(
+	suite *utils.TestSuite,
+	valAddr sdk.ValAddress,
+	signerName string,
+	signerAccount *dclauthtypes.Account,
+	info string,
+) (*sdk.TxResponse, error) {
+	msg := validatortypes.NewMsgProposeDisableValidator(suite.GetAddress(signerName), valAddr, info)
+	return suite.BuildAndBroadcastTx([]sdk.Msg{msg}, signerName, signerAccount)
+}
+
+func ApproveDisableValidator(
+	suite *utils.TestSuite,
+	accAddr sdk.ValAddress,
+	signerName string,
+	signerAccount *dclauthtypes.Account,
+	info string,
+) (*sdk.TxResponse, error) {
+	msg := validatortypes.NewMsgApproveDisableValidator(suite.GetAddress(signerName), accAddr, info)
+	return suite.BuildAndBroadcastTx([]sdk.Msg{msg}, signerName, signerAccount)
+}
+
+// Common Test Logic
+
+//nolint:funlen
+func ValidatorDemo(suite *utils.TestSuite) {
+	// // Jack, Alice and Bob are predefined Trustees
+	// jackName := testconstants.JackAccount
+	// jackKeyInfo, err := suite.Kr.Key(jackName)
+	// require.NoError(suite.T, err)
+	// jackAccount, err := test_dclauth.GetAccount(suite, jackKeyInfo.GetAddress())
+	// require.NoError(suite.T, err)
+
+	// aliceName := testconstants.AliceAccount
+	// aliceKeyInfo, err := suite.Kr.Key(aliceName)
+	// require.NoError(suite.T, err)
+	// aliceAccount, err := test_dclauth.GetAccount(suite, aliceKeyInfo.GetAddress())
+	// require.NoError(suite.T, err)
+
+	// bobName := testconstants.BobAccount
+	// _, err = suite.Kr.Key(bobName)
+	// require.NoError(suite.T, err)
+
+	// // Register new Vendor account
+	// vid := int32(tmrand.Uint16())
+	// nodeAdminName := utils.RandString()
+	// nodeAdminAcc := test_dclauth.CreateAccount(
+	// 	suite,
+	// 	nodeAdminName,
+	// 	dclauthtypes.AccountRoles{dclauthtypes.NodeAdmin},
+	// 	vid,
+	// 	aliceName,
+	// 	aliceAccount,
+	// 	jackName,
+	// 	jackAccount,
+	// 	testconstants.Info,
+	// )
+	// nodeAdminAddr, err := sdk.AccAddressFromBech32(nodeAdminAcc.Address)
+	// require.NoError(suite.T, err)
+	// validatorAddr := sdk.ValAddress(nodeAdminAddr)
+	// _, err = CreateValidator(suite, validatorAddr, nodeAdminName, nodeAdminAcc, nil, "test123")
+	// require.NoError(suite.T, err)
+
+	// // Query all proposed disable validators
+	// proposedDisableValidators, err := GetProposedValidatorsToDisable(suite)
+	// require.NoError(suite.T, err)
+	// require.Equal(suite.T, 0, len(proposedDisableValidators))
+
+	// // Query unknown disable validator
+	// _, err = GetDisabledValidator(suite, sdk.ValAddress(testconstants.ValidatorAddress1))
```

Comment:
This conversion is incorrect (here and below).

`ValidatorAddress1` is a Bech32 `cosmosvaloper` address. To convert it to `ValAddress` the function `ValAddressFromBech32` should be used.